src/renderer/Components/MyPackages/index.jsx

```diff
@@ -134,13 +135,36 @@ function MyPackages() {
                       {` "${order.sourcePath}" 로`}
                     </span>
                   )}
+                  <span className="font-bold text-blue-700">
+                    {` ${order.action}`}
+                  </span>
                 </div>
               ))}
             </div>
-            <div className="mt-6 flex justify-between text-right">
-              {/* TODO: 추후에 링크 기능 추가 */}
-              <p className="text-sm text-gray-400">
-                일련번호: {userPackage.serialNumber}
+            <div className="mt-6 flex flex-col justify-between text-left">
+              <p className="mb-2 text-sm text-gray-500">
+                <span className="font-bold">링크: </span>
+                <span
+                  className="cursor-pointer hover:font-bold"
+                  onClick={() => {
+                    navigator.clipboard.writeText(
+                      `electron-deliorder://open?packageId=${userPackage.serialNumber}`,
+                    );
+                  }}
+                >
+                  {`electron-deliorder://open?packageId=${userPackage.serialNumber}`}
+                </span>
+              </p>
+              <p className="text-sm text-gray-500">
+                <span className="font-bold">일련번호: </span>
+                <span
+                  className="cursor-pointer hover:font-bold"
+                  onClick={() =>
+                    navigator.clipboard.writeText(userPackage.serialNumber)
+                  }
```

Comment:
해당 부분을 위에 PackagePreview 에 있는 해당 함수를 유틸함수화 해서 받으시거나 prop 으로 받으셔서 사용하시면 재사용 할 수 있을 것 같습니다.😁
```js
 const copyToClipboard = (content) => {
    navigator.clipboard.writeText(content);
 };
```

 ```suggestion
                   onClick={() => copyToClipboard(userPackage.serialNumber)}
```

Reply:
유틸함수화 처리해보겠습니다!